misc.go

```diff
@@ -3,6 +3,8 @@ package commonservice
 import (
 	"context"
 	"time"
+
+	_ "github.com/golang/mock/mockgen/model"
```

Comment:
Shouldn't it be in a _test.go file?

Reply:
oh it is indeed cleaner ! I changed it !